src/phase0/state_transition/epoch_processing.rs

```diff
@@ -202,8 +202,13 @@ pub fn process_eth1_data_reset<
         MAX_VALIDATORS_PER_COMMITTEE,
         PENDING_ATTESTATIONS_BOUND,
     >,
-    _context: &Context,
+    context: &Context,
 ) -> Result<(), Error> {
+    let next_epoch = get_current_epoch(state, context) + 1;
+
+    if next_epoch % context.epochs_per_eth1_voting_period.into() == 0 {
```

Comment:
i don't see a reason to not strengthen the typing here on `epochs_per_eth1_voting_period` and then dropping the `into` call...

the diff will look something like this:

https://gist.github.com/ralexstokes/1430507703ed1cd00447113eefcb428f